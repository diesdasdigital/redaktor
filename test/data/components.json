{
  "default": {
    "required": {
      "title": "Components",
      "pageType": "components"
    },
    "fields": {},
    "components": [
      {
        "type": "headline",
        "value": "Hello Redaktor!"
      },
      {
        "type": "paragraph",
        "value": "This is our hello world."
      }
    ]
  }
}
