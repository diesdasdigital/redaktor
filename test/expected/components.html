<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Components</title>
  <meta name="viewport" content="width=device-width,initial-scale=1" />
</head>
<body>
  <h1>Hello Redaktor!</h1>
  <p>This is our hello world.</p>
</body>
</html>